packages/site/site-react/src/graphQLFetch/graphQLFetch.ts

```diff
@@ -0,0 +1,123 @@
+type Fetch = typeof fetch;
+
+export type BlockPreviewData = {
+    includeInvisible: boolean;
+};
```

Comment:
This type is new and is a copy of SitePreviewData which I removed from site-react. However, I believe this is better because it's used by `convertPreviewDataToHeaders` which in turn is used by `useBlockPreviewFetch`

Reply:
The same type is used for the site preview as well. Could we call it PreviewData and re-export it in site-nextjs?

Reply:
`convertPreviewDataToHeaders` is used by Site-Preview AND Block-Preview. That means `SitePreviewData` (as it was before) was strictly speaking wrong, but `BlockPreviewData` is also incorrect.

But that is just naming of two equal object types that include a single boolean.

I'd suggest to keep the duplicate but rename it to `PreviewData`. (Could also be just a includeInvisible boolean, but that would be an incompatible change)

Reply:
Alright: 3247ae14eec72f7049d5d7ab3b34a08d3696dfd7

I kept the SitePreviewData export but marked it as deprecated

Reply:
> Since it's a new package, can't we remove it altogether? Or do you want to enable a drop-in replacement in existing projects?

We could of course remove it. But up until now I didn't make any breaking changes API-wise compared to cms-site. So why start now?

I'd also like a drop-in replacement so I can easily test the new library in a real project